### PR TITLE
Added multilanguage

### PR DESCRIPTION
**Pull request by Fourmeta**

### CODE REVIEW

File: iogt/templates/base.html [hunk 1]
```diff
@@ -22,17 +22,19 @@
...
 
-        {# Global stylesheets #}
-        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-        <link rel="stylesheet" type="text/css" href="{% static 'css/iogt.css' %}">
-        <link rel="stylesheet" type="text/css" href="{% static 'css/header.css' %}">
-        <link rel="stylesheet" type="text/css" href="{% static 'css/footer.css' %}">
-        <link rel="stylesheet" type="text/css" href="{% sass_src 'css/sectionStyles.css' %}">
+    {# Global stylesheets #}
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (ehmadzubair) [inline iogt/templates/base.html:27]: We should host this locally. This will incur data cost to the user.

File: home/templatetags/home_tags.py [hunk 2]
```diff
@@ -55,6 +56,15 @@ def render_questionnaire_list(questionnaire):
...
+
 @register.inclusion_tag('home/tags/sub_sections.html')
 def render_sub_sections_list(sub_sections):
-    return {'sub_sections': sub_sections}
\ No newline at end of file
+    return {'sub_sections': sub_sections}
+
+
+@register.simple_tag
+def locale_set(locale, url):
```
Comment (ehmadzubair) [inline home/templatetags/home_tags.py:66]: Are we using this somewhere?

File: home/templatetags/home_tags.py [hunk 3]
```diff
@@ -55,6 +56,15 @@ def render_questionnaire_list(questionnaire):
...
+    for item in LANGUAGES:
+        code = item[0]
+        url = url.replace(f"/{code}/", "")
```
Comment (ehmadzubair) [inline home/templatetags/home_tags.py:69]: I don't understand what this is doing. Does overwriting the value of the url in the loop help us somehow?

File: iogt/static/css/iogt.css [hunk 4]
```diff
@@ -339,9 +339,9 @@ input:checked + .cust-check__title {
     border-bottom-right-radius: 10px;
     height: 100%;
     left: 0;
-    position: absolute;
+    /*position: absolute;*/
```
Comment (ehmadzubair) [inline iogt/static/css/iogt.css:342]: Better to remove this than comment

File: iogt/static/css/iogt.css [hunk 5]
```diff
@@ -339,9 +339,9 @@ input:checked + .cust-check__title {
...
     transition: all .45s;
-    width: 100%
+    /*width: 100%*/
```
Comment (ehmadzubair) [inline iogt/static/css/iogt.css:344]: Better to remove this than comment

**Review by ehmadzubair — CHANGES_REQUESTED** [review]

**Comment by ehmadzubair** [comment]
@Fourmeta 
There are a lot of formatting changes that have made their way in the diff. If they have to be done, they have to be done as a one-off exercise for the whole codebase. Can you revert them?

**Comment by dannyparsons** [comment]
replaced by #228

**Pull request closed without merging**